src/test/java/bio/terra/service/filedata/FileMetadataUtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package bio.terra.service.filedata;
+
+import static bio.terra.service.filedata.FileMetadataUtils.ROOT_DIR_NAME;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.common.category.Unit;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Category(Unit.class)
+public class FileMetadataUtilsTest {
+  @Autowired FileMetadataUtils fileMetadataUtils;
```

Comment:
I don't see why this needs to be autowired; does it it depend on an injectable value? Otherwise it would be quicker to instantiate a copy here directly and not use `SpringRunner`.

Looking at `FileMetadataUtils` I don't see why it's a `@Component` and not a class with static methods, like other Utils classes.

Reply:
Indeed! I've changed it to be static!